EE_API_automation/pytest/src/test_planner.py

```diff
@@ -672,7 +689,8 @@ def test_edit_workitem_desc(self):
             r = req.patch(url, headers=request_detail.headers_default, json=f)
             ##Analyze the response
             assert r.status_code == 200
-            desc_rendered = str(helpers.extract_value("data.attributes.\"system.description.rendered\"", r))
+            assert helpers.extract_value("data.attributes.\"system.description.markup\"", r) == "Markdown"
+            desc_rendered = str(helpers.extract_value("data.attributes.\"system.description.rendered\"", r)).strip()
```

Comment:
There should be an assert after this line. Where are we checking the `desc_rendered` value?

Reply:
line 694 asserts the check @jarifibrahim 

assert wi_new_desc in desc_rendered